docs/config.md

```diff
@@ -782,6 +808,78 @@ The base64 encoded protobuf describing (and containing) the node's private key.
 
 Type: `string` (base64 encoded)
 
+## `Internal`
+
```

Comment:
@aschmahmann @guseggert added the note that sets the tone for "pls dont touch this unless you really know what you are doing"  – feel free to adjust language if its too harsh :) 

```suggestion
## `Internal`

This section includes internal knobs for various subsystems to allow advanced users with big or private infrastructures to fine-tune some behaviors without the need for recompiling go-ipfs.  

**Be aware that making informed change here requires in-depth knowledge and most of users should left this untouched.** 

```

Reply:
Thanks, updated my diff to include fixes + note about breaking changes.